### DISCUSSION TITLE
FCPX / Final Cut Pro support (Scatter)

### DISCUSSION THREAD

**Discussion by danielrdotcom** [videovillage/community#160] [Ideas]
Any chance there's a version for FCPX in the works? That's my only holdup. I do some editing in Resolve and some in FCPX, but this would be great if I didn't have to round trip from resolve to FCPX to use Scatter for edits in FCPX. Thank you so much for having such a great product, just hoping to see this one more step.

**Comment by gregcotten** [comment]
Not currently in the works, but it might be something we port to in the future!

**Comment by Psychilles** [comment]
Daniel seems to be talking only about Scatter. But Filmbox is what I want in FCP. That would be I don't have to use Resolve for 90% of the work I do.

**Reply by danielrdotcom** [reply]
Filmbox would be cool too, but definitely was talking about Scatter. Grading is nicer in resolve, but editing is much more comfortable (and faster) for me on FCPX.

**Reply by Psychilles** [reply]
I also prefer grading in resolve of course. But with most of the work I do there’s no time for edit lock. And I would prefer to only work in FCP for those projects.

**Comment by wilg** [comment]
Shipped in Scatter 2.7.2: https://videovillage.com/scatter/releasenotes
